scripts/update-client.sh

```diff
@@ -47,6 +47,13 @@ if [[ -z ${GEN_ROOT:-} ]]; then
     GEN_ROOT="${TEMP_FOLDER}/gen"
     echo ">>> Cloning gen repo"
     git clone --recursive https://github.com/kubernetes-client/gen.git "${GEN_ROOT}"
+    # pick https://github.com/kubernetes-client/gen/pull/97
+    cd "${GEN_ROOT}"
+    git remote add tomplus https://github.com/tomplus/gen.git
+    git fetch tomplus
+    # somehow I couldn't get a clean cherrypick of the commit
```

Comment:
We need to remind ourselves about removing these lines in the future

Reply:
add a note in https://github.com/kubernetes-client/python/pull/931#issue-307594451